cdap-tms/src/test/java/co/cask/cdap/messaging/store/MessageTableTest.java

```diff
@@ -52,18 +52,36 @@ public void testSingleMessage() throws Exception {
       table.store(entryList.iterator());
       byte[] messageId = new byte[MessageId.RAW_ID_SIZE];
       MessageId.putRawId(0L, (short) 0, 0L, (short) 0, messageId, 0);
-      CloseableIterator<MessageTable.Entry> iterator = table.fetch(topicId, new MessageId(messageId), true, 50, null);
-      Assert.assertTrue(iterator.hasNext());
-      MessageTable.Entry entry = iterator.next();
-      Assert.assertArrayEquals(Bytes.toBytes(payload), entry.getPayload());
-      Assert.assertFalse(iterator.hasNext());
-      iterator = table.fetch(topicId, 0, 50, null);
-      entry = iterator.next();
-      Assert.assertArrayEquals(Bytes.toBytes(payload), entry.getPayload());
-      Assert.assertFalse(iterator.hasNext());
+
+      try (CloseableIterator<MessageTable.Entry> iterator = table.fetch(topicId,
+                                                                        new MessageId(messageId), false, 50, null)) {
+        // Fetch not including the first message, expect empty
+        Assert.assertFalse(iterator.hasNext());
```

Comment:
I am a little confused about the usage of inclusive here. So I thought inclusive meant that the messages should include that messageId and onwards. But say there is no message with that messageId, which means subsequent messages (including) the one after that should be fetched. 

Basically, does inclusive boolean same as includeFirst (or !skipFirst) ?